sqrl-pytorch/pytorch-cicd-test.sqrl

```diff
@@ -0,0 +1,10 @@
+IMPORT target.*;
+
+data := SELECT * FROM (VALUES
```

Comment:
I think we have to use data from a file to work around the current limitation of it trying to reduce static function calls. Or does this work now?